src/views/topics/TopicDetailView.vue

```diff
@@ -85,11 +87,29 @@ const breadcrumbLinks = computed(() => {
   return breadcrumbs
 })
 
-const tabTitles = [
-  { title: 'Données', tabId: 'tab-0', panelId: 'tab-content-0' },
-  { title: 'Discussions', tabId: 'tab-1', panelId: 'tab-content-1' },
-  { title: 'Réutilisations', tabId: 'tab-2', panelId: 'tab-content-2' }
-]
+const tabTitles: { title: string; tabId: string; panelId: string }[] = []
+if (pageConf.resources_tabs.datasets.display === true) {
```

Comment:
```suggestion
if (showDatasets === true) {
```
Maybe even move the `=== true` in `showXXX`, so the `if` here the `v-if` below are guaranteed to work the same way?